### DISCUSSION TITLE
Initialising Radiation Transport Systems

### DISCUSSION THREAD

**Discussion by hwhitehead** [PrincetonUniversity/athena#605] [Q&A] [answered]
Would anyone be able to provide information as to how best to initialize systems with radiation transfer? 

If I wished to study the evolution of a disc of hot gas, do the initial conditions need to include the details of the radiation field, or will the code initialize the field from the gas temperature?

Can any advice be given on reasonable numbers of angles to use with RT? If I was primarily focused on the vertical energy flux, would setting `nzeta = 1` and `npsi = 0` result in solving only for the vertical radiation flux?

**Comment by yanfeij** [comment]
It is recommended to read the documentation and try out some of the test
problems first:

https://github.com/PrincetonUniversity/athena/wiki/Non-relativistic-Radiation-Transport

On Tue, Jul 23, 2024 at 9:53 AM Henry Whitehead ***@***.***>
wrote:

> Would anyone be able to provide information as to how best to initialize
> systems with radiation transfer?
>
> If I wished to study the evolution of a disc of hot gas, do the initial
> conditions need to include the details of the radiation field, or will the
> code initialize the field from the gas temperature?
>
> Can any advice be given on reasonable numbers of angles to use with RT? If
> I was primarily focused on the vertical energy flux, would setting nzeta
> = 1 and npsi = 0 result in solving only for the vertical radiation flux?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PrincetonUniversity/athena/discussions/605>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCWEJ3VUFOENVFZOHFOYGLZNZN67AVCNFSM6AAAAABLKPVZIWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHE3DQNRWGY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Best,

Yan-Fei Jiang
Associate Research Scientist,
Flatiron Institute,
160 Fifth Avenue
New York, NY 10010

**Reply by hwhitehead** [reply]
Perhaps you could help me with some of the definitions in the documentation. I believe there are four opacities that are used in the code associated with scattering, Rosseland mean, Planck mean and "radiation energy weighted mean".  I could not find this last opacity in the code papers referenced in the documentation, do you know how it is defined?

Can any information be given as to the "default" opacity settings, are they set by free-free absorption? I see an option for enrolling a custom opacity function, but I do not believe there are any examples of its use in the wiki or example pgen files. Are there any examples of its usage available? The only examples I can see feature setting all non-scattering opacities to constants.

**Comment by yanfeij** [comment, marked as answer]
I added an example near the end of the page:
https://github.com/PrincetonUniversity/athena/wiki/Non-relativistic-Radiation-Transport

Hope it is helpful.

On Wed, Jul 24, 2024 at 7:48 AM Henry Whitehead ***@***.***>
wrote:

> Perhaps you could help me with some of the definitions in the
> documentation. I believe there are four opacities that are used in the code
> associated with scattering, Rosseland mean, Planck mean and "radiation
> energy weighted mean". I could not find this last opacity in the code
> papers referenced in the documentation, do you know how it is defined?
>
> Can any information be given as to the "default" opacity settings, are
> they set by free-free absorption? I see an option for enrolling a custom
> opacity function, but I do not believe there are any examples of its use in
> the wiki or example pgen files. Are there any examples of its usage
> available? The only examples I can see feature setting all non-scattering
> opacities to constants.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PrincetonUniversity/athena/discussions/605#discussioncomment-10136874>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCWEJ6QS5OHXKTYOHY63XTZN6ICHAVCNFSM6AAAAABLKPVZIWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJTGY4DONA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


-- 
Best,

Yan-Fei Jiang
Associate Research Scientist,
Flatiron Institute,
160 Fifth Avenue
New York, NY 10010

**Reply by hwhitehead** [reply]
Much appreciated, can I just check if there is a typo in the last line: is `sigma_a` the Rosseland opacity and `sigma_s` the scattering opacity, or is `sigma_a + sigma_s` the Rosseland opacity?

**Comment by hwhitehead** [comment]
Extra information about the NR-RT module can be found in this [thread](https://github.com/PrincetonUniversity/athena/discussions/635).
